operations_on_datastructures/linked_list_right_rotation.cpp

```diff
@@ -0,0 +1,210 @@
+/**
+ * @file
+ * @brief Rotate a linked list 'n' number of times.
+ * @details
+ * The linked list is rotated to right side by one step 'n' times.
+ **/
+
+#include <cassert> /// for assert
+#include "bits/stdc++.h"
```

Comment:
Please do not use `bits/stdc++.h` as it includes lots of other functions and unused variables that slow the compilation process.
It's best to include the headers/libraries one by one separately to ensure you're only loading headers/libraries you will need or use. 🙂

Reply:
Done.